### ISSUE TITLE
[Not an issue] I thought I should share my tip on how to restart the bot when you are away from ur 3ds

### ISSUE THREAD

**Issue by Naturally-Delicious** [olliz0r/Ledybot#46]
Sorry if this is common knowledge, but this incredible helped me when I am away from my 3DS and I am running a giveaway with Ledybot. Now, lets say you have a giveaway on-going, and you have to step out of the house for a bit. Ledybot now crashes when you're out, leaving ur participants waiting for a long time. This is bad, and since you are not near your 3ds, you cannot restart the bot. Well, I have figured out a way around this. I think it'll help others who have to go out a lot and can't monitor their 3ds for when it disconnects from the internet and such.

REQUIREMENT:
Device that can have chrome remote desktop app (ex. your smartphone)
Host device (ur computer) that can run Chrome remote desktop app
Internet connection on 2nd device (not host)
PKMN-NTR

Recommened that your 2nd device have mobile data. You'll be able to do it anywhere then. (don't worry, chrome remote desktop doesn't take that much data as long as you're not on it for hours)

First, you want to set up your host device and the 2nd device with Chrome remote desktop. Chrome remote desktop allows you to remote control your devices. (similar to input redirection!) useful just for this purpose. There are tutorials on how to set it up, so go do that and then come back here.

Once your host device and 2nd device are all set up, all you need to do is frequently check for an error message. The error message means Ledybot stopped and is not trading anymore (You check this by connecting your 2nd device to your host device with chrome remote desktop)

When that error comes up, on your 2nd device, connect to your host device through chrome remote desktop. You are not controlling the device that hosts ledybot.  disconnect from Ledybot and then reconnect. Then start. Sometimes, the bot just messes up with inputs or something and it will continue as normal. But, sometimes, the 3DS is disconnected from the internet, and you will have to reconnect it. You will know since after a while you have the bot running, an error will pop up saying "An error has occurred". This 100% means your 3DS  is disconnected from internet. Ledybot will try to go in recovery.

Usually, when your 3DS disconnects, a screen with an error code pops up. Pressing any button will remove the screen and bring you back to main menu. Since Ledybot presses "B" as recovery proccess, that screen will go away and bring you to the main menu. I found out that apart of Ledybot's recovery proccess, it'll click "Trade" and "GTS" for you. You'll now be at the screen where it says "Do you want to connect to the internet?" and blah blah blah. This is when Ledybot brings up the error since it doesn't know what to do (I think? I just know it gives an error when it's at this part) For this part, you only need to Press "A", a few simple button presses. PKMN-NTR will be able to do this easily with remote control.

Dismiss the error and Disconnect from Ledybot, and open up PKMN-NtR and connect your 3DS. Go to Tools, then remote control. Look for the "A" button and repeatedly press that. I usually do it for one minute just to be safe. Pressing "A" will get you pass the "connecting to the Internet" section and bring you to the GTS. Disconnect from PKMN-NTR and connect your 3DS to Ledybot. Press Connect, start, and you should be back in business! It may go in recover if you pressed "A" lot, but that shouldn't be a major concern.

A few problems that may arise:

Since you are not looking at your 3DS screen, you have *no* idea what is happening. You may arrive at the wrong screen, and the pattern will be broken. It is just pure luck to be honest. I have had very high success rate with it though, so the chance of arriving at the wrong screen is low.

You may get the "Power off your 3DS" screen, which happenes when your internet cuts off *just* as the bot is going to send a Pokemon. Rare, but it can happen. At this point, you are screwed as you have to power off the 3DS.

The program cannot connect to your 3DS. Since you cannot use Ledybot or PKMN-NTR, you cannot use remote control to get past the screens. You are screwed here too.

Yes, this tutorial looks huge, but once you do it and know what you're going for, it only takes about 5-10 minutes. (TBH it is just padded with why it works) Good to do when you're on a lunch break at work or something.

So basically, the proccess is this:

1. Disconnect and reconnect Ledybot. Press start and wait for an error message to appear. If none appears, that means Ledybot screwed with the inputs or something and will continue like normal.
2. Disconnect Ledybot, connect to PKMN-NTR and go to remote control. Press A repeatedly for about a minute or so. (Give enough time to connect to internet and yeah)
3. When you think you're at GTS, disconnect PKMN-NTR and connect Ledybot and start it like normal. 

again... I am sorry if there is a better way or it is common knowledge, but I am happy to share this since I go out a lot and usually have ledybot offline for hours when a giveaway is happening. I hope anyone in similar situations will find this helpful. :)

**Comment by k20stitch** [comment]
Not really sure where this belongs. While it's great information, it really only pertains to people who leave the bot running when they're away whether it be a stream or something similar.  Normal operation of this bot assumes you're sitting in front of it.   If the author feels that it's worthy enough, maybe it can be added to the read me, otherwise this should just be closed to reduce clutter.

**Comment by Naturally-Delicious** [comment]
I don't know where else to put it, sorry if it feels irrelevant. When I
found out about this trick though, it helped me so much with giveaways, so
I thought I'd share.
